splunkforwarder/elx/init.sls

```diff
@@ -11,6 +11,20 @@
 {%- from tpldir ~ '/map.jinja' import splunkforwarder with context %}
 
 {%- for port in splunkforwarder.client_out_ports %}
+  {%- if salt.grains.get('osmajorrelease') == '7'%}
+    {%- set FwZone = salt.firewalld.default_zone() %}
+Allow Splunk Mgmt Outbound Port {{ port }}:
+  module.run:
+    - name: 'firewalld.add_port'
```

Comment:
No. Already tested it under that condition. Even if you run it a thousand times (speculating, obvs.) it doesn't bomb. It reports having made a change, however, if the exception's already there, it simply drops any would-be dupes (friendlier than bare `iptables` in that regard, at least).